src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -919,6 +919,18 @@
     <value>Maximized focus</value>
     <comment>An option to choose from for the "launch mode" setting. Opens the app maximized and in focus mode.</comment>
   </data>
+  <data name="Globals_LaunchModeMaximizedFullscreen.Content" xml:space="preserve">
+    <value>Maximized full screen</value>
+    <comment>An option to choose from for the "launch mode" setting. Opens the app maximized and in full screen.</comment>
+  </data>
+  <data name="Globals_LaunchModeFullscreenFocus.Content" xml:space="preserve">
+    <value>Full screen focus</value>
+    <comment>An option to choose from for the "launch mode" setting. Opens the app in full screen and in focus mode.</comment>
+  </data>
+  <data name="Globals_LaunchModeMaximizedFullscreenFocus.Content" xml:space="preserve">
+    <value>Maximized full screen focus</value>
+    <comment>An option to choose from for the "launch mode" setting. Opens the app maximized in full screen and in focus mode.</comment>
+  </data>

```

Comment:
Do we actually need these since we're removing them in Launch.cpp?

Reply:
Technically this was the way that had the least amount of code. Basically the macro currently used attempts to look up every possible enum value in the localization files and so it would crash if it didn't exist. I could've written a different macro/manually write the code so it didn't try the values being filtered out. But since I added the text for the one option I was just lazy and left it in.